### ISSUE TITLE
Browser support

### ISSUE THREAD

**Issue by g-harel** [SOEN341-Group1/KoalaCal#124]
I would like to discuss browser support with everyone. Easiest thing would be latest chrome, but it would make more sense to support a bit more than that. 

@RemusCM @jaslatendresse @eCom0327 @RanjuShetty

**Comment by RanjuShetty** [comment]
http://www.digitaltrends.com/computing/best-browser-internet-explorer-vs-chrome-vs-firefox-vs-safari-vs-edge/

I think we can use either chrome or firefox ..

**Comment by g-harel** [comment]
So not both ?

**Comment by RanjuShetty** [comment]
Yes that would be a good idea, but are we taking into consideration the development plug-in aspect or the interface output integration?

**Comment by g-harel** [comment]
What do you mean by plug-in ?
